<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Importer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="browsePushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="directoryLabel">
       <property name="text">
        <string>Search Path:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="viewModeSelector">
       <item>
        <property name="text">
         <string>Category List</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name List</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File Tree</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="metaFileLineEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pathLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="importsFileLabel">
       <property name="text">
        <string>Imports File:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="metaFileLabel">
       <property name="text">
        <string>Model Mapping File: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="importsFileLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="viewModeLabel">
       <property name="text">
        <string>View Mode:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="errorsFileLabel">
       <property name="text">
        <string>Errors File:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="errorsFileLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="updatePushButton">
       <property name="text">
        <string>Generate Model Mapping File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reloadPushButton">
       <property name="text">
        <string>Reload Model Mapping File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
